docs/website/docs/dlt-ecosystem/verified-sources/workable.md

```diff
@@ -236,7 +236,7 @@ To create your data pipeline using single loading and [incremental data loading]
 1. To load data from a specific date, including dependent endpoints:
 
    ```py
-   load_data = workable_source(start_date=datetime(2022, 1, 1), load_details=True)
+   load_data = workable_source(start_date=DateTime(2022, 1, 1), load_details=True)
```

Comment:
Here, the type is correct, but now it's not clear where `DateTime` is coming from. Maybe it'd be better to explicitly use `pendulum.DateTime`?